modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the {redfish-operator}.
+* Create a `BMCEventSubscription` CR for the BMC Redfish hardware.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. Create the `HardwareEvent` custom resource (CR) entry in the `PolicyGenTemplate`:
+
+.. Save the following YAML in the `hw-event.yaml` file:
++
+[source,yaml]
+----
+apiVersion: event.redhat-cne.org/v1alpha1
+kind: HardwareEvent
+metadata:
+  name: openshift-hw-events
+  namespace: openshift-hw-events
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  nodeSelector: {}
+  transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
+  logLevel: "trace"
+----
+
+.. Create the `HardwareEvent` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f hw-event.yaml
+----
+
+. Update the `spec.sourceFiles` entry in the `PolicyGenTemplate` with `HadrwareEvent` CR file that configures the AMQ host to the `hardware-events-config`.
++
+[source,yaml]
+----
+- fileName: hw-events.yaml
+  policyName: "hardware-events-config"
+  metadata:
+    name: openshift-hw-events
+    namespace: openshift-hw-events
+    annotations:
+      ran.openshift.io/ztp-deploy-wave: "10"
+  spec:
+    nodeSelector: {}
+    transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
+    logLevel: "trace"
```

Comment:
I don't think this step is needed since it is already created in the ztp repo. We only need to add the file in spe.sourceFiles in PolicyGenTemplate.